.github/ISSUE_TEMPLATE/translation.yml

```diff
@@ -0,0 +1,35 @@
+name: Translation
+description: Request a translation
+title: ":twisted_rightwards_arrows: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate you taking the time to fill out this translation request!
+  - type: input
+    id: word
+    attributes:
+      label: What word do you want to translate (or see translated)?
+    validations:
+      required: true
+  - type: input
+    id: language
+    attributes:
+      label: What langauge would you like to this word translated to?
+    validations:
+      required: true
+  - type: checkboxes
+    id: assign-to-me
+    attributes:
+      label: Would you like to take ownership of defining this word?
+      description: By checking this, you'll be assigned this issue and we'll work with you to get the definition added should it be accepted!
```

Comment:
Not sure how best to frame this, but it would be helpful to ask if they have folks who could read their translation if so. It's sometimes hard to source people to review languages.

Reply:
Interesting! Let me think about this one!

Reply:
One way I might frame it (feel free to use/not use) is:

> **Do you have any fluent colleagues who could review your translations?** Translations will always be shared with our  community to review, but we recognise that many languages are not yet widely represented. We always want to ensure all entries are peer-reviewed by as many folks as possible.